src/modules/users/users.ctrl.js

```diff
@@ -143,6 +142,71 @@ const deleteRel = async ({ services }, body) => {
   return data;
 };
 
+// Other User Controllers
+const WaitingForAccountConfirmation = async ({ services }, body) => {
+  MapConfirmAccount.set(body.email, body);
+
+  const token = jwt.createToken({ email: body.email });
+
+  services.email.send(
+    body.email,
+    "Confirmación de Cuenta",
+    services.template.confirmEmail(body.userName, "http://localhost:5173/cuenta/" + token + "/confirmacion")
+  );
+
+  return { data: "Se ha enviado un mensaje a " + body.email + " para confirmar tu cuenta" };
+};
+
+const recoveryAccount = async ({ services }, body) => {
+  let data = await services.neo4j.session.run(logInQuery(body));
```

Comment:
This is always going to throw an error, almost all the queries filter out the deleted accounts